.github/workflows/build.yml

```diff
@@ -15,6 +15,7 @@ jobs:
       - run: go build -o target/ ./cmd/...
       - name: Run Go tests and generate in json format
         run: go test -json -v ./... | tee report.json
+        continue-on-error: true
```

Comment:
but still fails right after publishing the report, right?

Reply:
no this all worked fine (i ran it based off your Pr and it got the right result)